packages/telegraf/extra/telegraf.conf

```diff
@@ -0,0 +1,130 @@
+[global_tags]
```

Comment:
this file should not been added to the package, right? my understanding is it should be part of the file https://github.com/dcos/dcos/blob/master/gen/dcos-config-win.yaml, right @jongiddy ?

Reply:
In the future the telegraf.conf will be moved to dcos-config-win.yaml